test/unit/rounding/rounding.cpp

```diff
@@ -123,4 +123,80 @@ namespace {
                     "cnl::shift_right test failed");
         }
     }
+
+    namespace tie_to_pos_inf_rounding {
+
+        namespace convert {
+            static_assert(
+                    identical(
+                            cnl::uint8{101},
```

Comment:
Compare this to the corresponding test of cnl::nearest_rounding_tag. I had to decrease the numerical range for this to compile. Maybe similar thing is happening on line 181?